content/docs/introduction/getting_started.md

```diff
@@ -224,10 +219,15 @@ Try graphing this expression.
 
 To record the time series resulting from this expression into a new metric
 called `job_service:rpc_durations_seconds_count:avg_rate5m`, create a file
-with the following recording rule and save it as `prometheus.rules`:
-
-```
-job_service:rpc_durations_seconds_count:avg_rate5m = avg(rate(rpc_durations_seconds_count[5m])) by (job, service)
+with the following recording rule and save it as `prometheus.rules.yaml`:
```

Comment:
We use .yml elsewhere.

I remain a fan of a `filename.rules`.
This is likely to get confused with `prometheus.yml`, a different example may be better (e.g. `service`).